gc/base/Configuration.hpp

```diff
@@ -136,6 +136,9 @@ class MM_Configuration : public MM_BaseVirtual
 	 * @param[in] env the current environment
 	 * @return void
 	 */
+	virtual void adjustGCThreadCountForCheckpoint(MM_EnvironmentBase* env) { adjustGCThreadCountOnCheckpoint(env); }
```

Comment:
This line is too long (>100 chars); it should be spread across multiple lines:
```suggestion
	virtual void
	adjustGCThreadCountForCheckpoint(MM_EnvironmentBase* env)
	{
		adjustGCThreadCountOnCheckpoint(env);
	}
```